.travis.yml

```diff
@@ -37,7 +37,7 @@ before_install:
   - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.6.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
   - mkdir -p $HOME/.kube $HOME/.minikube
   - touch $KUBECONFIG
-  - sudo minikube start --vm-driver=none --kubernetes-version=v1.15.7
+  - sudo minikube start --profile=minikube --vm-driver=none --kubernetes-version=v1.18.1
```

Comment:
We need to add the `before_install` step to install the conntrack pkg to use the latest release of k8s
```
before_install:
- sudo apt-get -qq -y install conntrack
```
Check here
https://github.com/openebs/cstor-csi/blob/b82d38349120403ebb63973d045ccac3cf16b383/.travis.yml#L20